### ISSUE TITLE
Session start time in stats that are sent to monitor

### ISSUE THREAD

**Issue by lukasz-glen** [golemfactory/clay#2438]
Every hour a node sends stats to the monitor. The stats that are sent are aggregated for a session, they have summary data from the beginning of the session. To calculate, for instance, how many subtasks were computed by a node in the last hour, we need to take a difference.
Unfortunately, there is no mark in stats that session was restarted. Add a timestamp of a session start to stats to enable distinction of sessions in the monitor.
Persist that timestamp in the monitor's database.

**Comment by RDProjekt** [comment]
Golem adds a session ID to every message it sends to the monitor (sessid in Redis and PostgreSQL) along with a timestamp. Session ID changes every time Golem is started, so based on the session ID and the first/last timestamps bound to it, we can identify the session start/end timepoints with an accuracy of 1h - this is the interval of taking snapshots, not sending out stats, which happens much more often.

Summing up - do we really need the session start timestamp, or are the current data enough for now?

**Comment by lukasz-glen** [comment]
As stated above
